<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IActionSelect</class>
 <widget class="QWidget" name="IActionSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>92</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="radioButton1">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButton4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
